blocks/search-results-list-block/features/search-results-list/default.jsx

```diff
@@ -55,13 +55,18 @@ SearchResultsListContainer.icon = "arc-list";
 
 SearchResultsListContainer.propTypes = {
 	customFields: PropTypes.shape({
-		searchContentConfig: PropTypes.contentConfig().tag({
-			group: "Configure Content",
-			label: "Display Content Info",
+		imageRatio: PropTypes.oneOf(["16:9", "3:2", "4:3"]).tag({
```

Comment:
Note: Adding a new custom field does not populate pages/templates that already have the block added to them with the `defaultValue` - the `imageRatio` would be undefined when the block is rendering